packages/paste-design-tokens/tokens/global/palette-color.yml

```diff
@@ -44,3 +44,34 @@ props:
   color-brand-highlight:
     value: "{!amaranth}"
     comment: "Twilio brand red"
+  # Data Viz Colors
+  color-data-viz-1:
+    value: "{!palette-blue-80}"
+    comment: Color used in a sequence for data vizualizations.
```

Comment:
vizualizations -> visualizations

Reply:
Also, what would you think of updating the description just a touch to: "Color used for data visualizations. Must be used in a sequence."

We're starting to work on refining our token definitions and having them all follow a similar format, and a lot of our other tokens use the first sentence for a definition and an optional second sentence for usage restrictions.

Reply:
Works for me!